systems/WikibaseRepo/01-Introduction_and_Goals.md

```diff
@@ -29,6 +29,10 @@ Wikibase Repository is part of [Wikibase](../../Introduction_and_Goals.md). Wiki
 
 - Edit [Terms](../../Glossary.md#term) and sitelinks of entities without using JavaScript in a browser.
 
+### Wikidata reader
+
+- Look up and explore facts about the world
```

Comment:
Similar to above. We avoid "facts".

Reply:
After reading [your comment on the phab ticket](https://phabricator.wikimedia.org/T273973#6826103): Should I remove this completely?

Reply:
I think I'd remove it, yeah.